lib/convertyaml.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import json
+import tempfile
+import yaml
+from pathlib import Path
+from subprocess import run
+
+
+"""
+python lib/convertyaml.py \
+    tenant \
+    --git https://gitlab.cee.redhat.com/gnecasov/container-errata-templates.git \
+    --branch main \
+    --path RHEL/XXXXX.yaml
+"""
+
+
+def read_parameters():
+
+    parser = argparse.ArgumentParser()
```

Comment:
missing release and previousrelease arguments. I know they are not used but they will passed